src/Previewer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+const StyledContainer = styled.div`
+    float:right;
+    width: 45%;
+    height: 400px;
+    padding: 12px 20px;
+    box-sizing: border-box;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    background-color: #f8f8f8;
+    resize: none;
+    border-radius: 20px;
+    padding: 0.25em;
+    margin: 2em;
+    margin-left: 25px;
+    margin-right: 25px;
+    align: center;
+    text-align: center;
+    font-family: arial;
+`;
+
+const StyledTitle = styled.h3`
+    text-align: center;
+`;
+
+const StyledPreviewer = styled.textarea`
+    width: 80%;
+    height: 70%;
+`;
+
+class Previewer extends React.Component {
```

Comment:
again, you can transform into an SFC if you don't have state

Reply:
something like this 
const Previewer = () => {
 return (
            <StyledContainer>
                <StyledTitle>Previewer</StyledTitle>
                <StyledPreviewer />
            </StyledContainer>
        )
}
But please google it, read about it and tell me why we should use a SFC over the class syntax.